### PR TITLE
LWIMIQ 12-11

### PR DESCRIPTION
**Pull request by jntullo**
cc @JPrause

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @jntullo 
@imtayadeway @syncrou @d-m-u can you each review if time permits.

**Comment by JPrause** [comment]
Yeh! LGTM,..and now I'm all excited for Christmas. 🎁 🎅 🤶 🦌

File: site/_posts/2017-12-11-bug-workshop.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Improved
+A simple update of an error message can go a long way in helping users to understand a problem. This [update](https://github.com/ManageIQ/manageiq/pull/16507) fixes a uniqueness check to return a message that matches what would be expected from a user in the UI. 
+
+It's always nice to have styling improved, and this [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/2961) updates Capacity & Utilization options to be more consistent with other UIs. The improvements look great! :heart_eyes:
+
+## Fixed
+In the Classic UI, Generic Object instances for Ansible Playbook services are now being [displayed](https://github.com/ManageIQ/manageiq-ui-classic/pull/2974).
+
+In the Service UI, angular is no longer being loaded too many times thanks to the removal of an unnecessary [dependency](https://github.com/ManageIQ/manageiq-ui-service/pull/1295).
```
Comment (imtayadeway) [inline site/_posts/2017-12-11-bug-workshop.md]: I'm wondering if angular should be capitalized?

**Comment by JPrause** [comment]
@cybette can you review and merge if it LGTY.

**Pull request merged**